1.0: in_tail_with_throttle: Add `<group>` directive description

Pranjal-Gupta2 opened the pull request:
Documentation for [fluent/fluentd#3535](https://github.com/fluent/fluentd/pull/3535)

Signed-off-by: Pranjal-Gupta2 <pranjal.gupta2@ibm.com>

Pranjal-Gupta2 committed:
in_tail_with_throttle: Add documentation for `<group>` directives

Signed-off-by: Pranjal-Gupta2 <pranjal.gupta2@ibm.com>

cosmo0920 reviewed (changes requested):
Could you rewrite to be corresponding to https://github.com/fluent/fluentd/pull/3535?
https://github.com/fluent/fluentd/pull/3535 is very reasonable for us but the documentation is also needed to ship into stable release.

Pranjal-Gupta2 committed:
Update `in_tail` with group directive

Signed-off-by: Pranjal-Gupta2 <pranjal.gupta2@ibm.com>

cosmo0920 reviewed (approved):
Seems good for me.

daipom commented on input/tail.md:
```diff
@@ -402,6 +412,80 @@ The `@log_level` option allows the user to set different levels of logging for e
 Refer to the [Logging](../deployment/logging.md) for more details.
 
+
+### `<group>` Section \(highly recommended\)
```
```suggestion
### `<group>` Section
```

I think this function is rather optional.
(Use when we want more detailed control than `read_bytes_limit_per_second`)

ashie replied:
Removed in c0a9c1e35276d9e33ec8280b6a33f6aa9c46c253

ashie committed:
in_tail: Remove `<group>` section in example config

In addition, remove `higly recommended` label for it.
The priority of this option depends on usage and purpose of users.

Signed-off-by: Takuro Ashie <ashie@clear-code.com>

daipom commented on input/tail.md:
```diff
@@ -17,6 +17,16 @@ It is included in Fluentd's core.
   <parse>
     @type apache2
   </parse>
+  <group>
+    rate_period 30s
+    <rule>
+      match {
+          namespace: /space1|space2|space2/,
+          podname: /app.*/,
+      }
+      limit 200
+    </rule>
+  </group>
```
For the same reason, I think it would be better to leave the simple configuration example here as it is
and have a separate configuration example for this feature.

daipom replied:
How about moving this example to just below the `<group>` Section and reverting this block?

ashie replied:
Just removed in c0a9c1e35276d9e33ec8280b6a33f6aa9c46c253
Since another example already exists just under `<group>`, I simply removed it.

daipom commented:
Thank you for the fix!

ashie commented:
I merge this since v1.15 has been released.
Thank you for your effort!
